src/cli.rs

```diff
@@ -1,19 +1,14 @@
-use std::path::PathBuf;
-
 use clap::{App, Arg};
 
 use crate::errors::*;
-use crate::filesystem::binding::Binding;
 use crate::filesystem::validation::{binding_validator, path_validator};
 use crate::filesystem::FileSystem;
 
 pub const DEFAULT_ROOTFS: &'static str = "/";
-pub const DEFAULT_CWD: &'static str = ".";
+pub const DEFAULT_CWD: &'static str = "/";
 
-pub fn parse_config() -> Result<(FileSystem, Vec<String>)> {
-    let mut fs: FileSystem = FileSystem::new();
-
-    let matches = App::new("proot-rsc")
```

Comment:
I wonder why this was `proot-rsc` before?

Reply:
I found it, it's this: https://github.com/proot-me/proot-rs/commit/d5a546d003ab710389af77ed005d7895f6cd6844#diff-b2812f19576dd53d0c35b107a322f58a00fce6977f4b5976e1961853982af3cc

Maybe this means `proot-rust-cli`? I think it will be better to keep it same as the project name (proot-rs)